skfda/exploratory/visualization/representation.py

```diff
@@ -323,3 +320,165 @@ def plot_scatter(fdata, chart=None, *, grid_points=None,
     _set_labels(fdata, fig, axes, patches)
 
     return fig
+
+def plot_with_gradient(fdata, chart=None, *, fig=None, axes=None,
```

Comment:
We will try to follow the scikit-learn visualization API, with *Display objects and class methods to create them.